### DISCUSSION TITLE
Openid configuration in Apache

### DISCUSSION THREAD

**Discussion by Supraja1989** [OpenIDC/mod_auth_openidc#821] [Q&A]
Hi,

   Am new to openid connect.Can anybody help me in configuring Apache with mod_auth_openidc for 2-5 applications hosting in Apache & redirecting to backend weblogic servers

Regards
Supraja G

**Comment by chrisinmtown** [comment]
This is a wide-open question, instead you probably should post what you have tried so far. Please note you can configure Open IDC separately in VirtualHost configuration blocks which might serve your need to handle multiple different applications.
